src/modules/EventAnnoucementModule.js

```diff
@@ -61,18 +61,18 @@ class EventAnnoucementModule extends Module {
                 return;
 
             const dmembed = new Discord.RichEmbed()
-                .setTitle(Translation.translate("module.eventannoucement.title." + name))
-                .setColor(0xbadc58)
-                .setDescription(Translation.translate("module.eventannoucement.description." + name, time.format("D. M. YYYY"), eventValues.title))
+                .setTitle((eventValues.type == "event") ? "👁 " + Translation.translate("module.eventannoucement.event.title." + name) : "👁 " + Translation.translate("module.eventannoucement.task.title." + name))
```

Comment:
You can do this. Change it that way.
```suggestion
                .setTitle("👁 " + (eventValues.type == "event" ? Translation.translate("module.eventannoucement.event.title." + name) : Translation.translate("module.eventannoucement.task.title." + name)))
```

Reply:
Anyways you can do this: Translation.translate("module.eventannoucement." + eventValues.type +".title." + name). Do these things in all references.